components/infobox/commons/infobox_series.lua

```diff
@@ -364,9 +322,43 @@ function Series:_createOrganizers(args)
 	return organizers
 end
 
+function Series:_getCategories(args)
+	local categories = {'Tournament series'}
+
+	for _, country in Table.iter.pairsByPrefix(args, 'country', {requireIndex = false}) do
+		table.insert(categories, self:_setCountryCategories(country))
+	end
+
+	Array.extendWith(categories, self:addTierCategories(args))
+
+	return categories
+end
+
+function Series:addTierCategories(args)
+	local categories = {}
+	local tier = args.liquipediatier
+	local tierType = args.liquipediatiertype
+
+	local tierCategory, tierTypeCategory = Tier.toCategory(tier, tierType)
+	local isValidTierTuple = Tier.isValid(tier, tierType)
+	table.insert(categories, tierCategory)
+	table.insert(categories, tierTypeCategory)
+
+	if not isValidTierTuple and not tierCategory and String.isNotEmpty(tier) then
+		table.insert(self.warnings, String.interpolate(INVALID_TIER_WARNING, {tierString = tier, tierMode = 'Tier'}))
+		table.insert(categories, 'Pages with invalid Tier')
+	end
+	if not isValidTierTuple and not tierTypeCategory and String.isNotEmpty(tierType) then
+		table.insert(self.warnings, String.interpolate(INVALID_TIER_WARNING, {tierString = tierType, tierMode = 'Tiertype'}))
+		table.insert(categories, 'Pages with invalid Tiertype')
+	end
+
+	return categories
+end
+
 function Series:_setCountryCategories(country)
 	if String.isEmpty(country) then
-		return ''
+		return
```

Comment:
iirc, this needs to return nil to not break the table.insert

Reply:
correct

Reply:
yeah, thx